docs/changes.rst

```diff
@@ -410,56 +392,41 @@ Enhancements
   consistency with ignoring the path on
   `CIMInstance.tocimxml()/tocimxmlstr()` (as part of fixing issue #1136).
 
-* Added unit test for WBEMServer class using pywbem_mock.  See the file
-  testsuite/test_wbemserverclass.py.  This test is incomplete today but tests
-  most of the main paths.
-
-* Revamped the logger configuration mechanism completely. This includes:
-  (1) Created 3 methods in WBEMConnection that allow pywbem logs to be
-  configured and activated.  These methods contain parameters for:
-  a. configuring the Python loggers for either/or/both the api and http
-  loggers. b. Setting the level of detail in the log output. c. Activating
-  each logger within WBEMConnection.
-  (2) Allow for the standard Python loggers to be used to configure logger
-  names that will be used by the pywbem loggers. This allows the pywbem
-  loggers to be compatible with user code that creates their specific logger
-  configurations.
-  (3) Eliminated the PyWBEMLogger class that was the original
-  logging setup tool in pywbem 0.11.0 since its use was incompatible with
-  using standard Python logging configuration methods to define loggers.
-  (see issue #859)
-  (4) Created a function in the _logging module that allows pywbem logging
-  to be defined by a single string input.
-  (5) ddition of a new property to WBEMConnection (conn_id) which is a
-  unique identifier for each WBEMConnection object and is part of each log
-  record. This allows linking logs for each WBEMConnection in the log.
-
-* Improved the complete pywbem documentation (Issue #1115).
-
 * Improved the handling of certain connection errors by retrying and by
   issuing user warnings instead of printing if debug. (Issue #1118).
 
 Bug fixes
 ^^^^^^^^^
 
-* Fixed issue in iterReferenceNames and IterAssociatiorNames where it was
-  not passing the IncludeQualifiers input parameter to the OpenReferenceNames.
-  This should not have been a significant issue since in general qualifiers
-  are not parts of instances.  Also changed code in IterQueryInstances were
-  parameters that are required by the called ExecQuery and OpenQueryInstances
-  were defined as NamedArguments where since they are required, the Name
-  component is not required.  This should not change operations except that
-  when we were mocking the methods, it returns sees the parameter as name=value
-  rather than value. See issue #833
+* Added `libxml2` operating system package as a dependency. It provides xmllint,
+  which is used for testing.
+
+* Fixed issue where `MOFCompiler.compile_str()` could not compile MOF that was
```

Comment:
I think that is compile_string. Actually we should chane name in mock to _string to match.

Reply:
I think it is already _string

Reply:
DONE